src/data/interfaces/StoreNames.ts

```diff
@@ -0,0 +1,16 @@
+import { Table } from 'dexie';
+import { Activity } from './Activity';
+import { Category } from './Category';
+import { Day } from './Day';
+
+export const DAY_STORE = 'days';
+export const ACTIVITY_STORE = 'activities';
+export const CATEGORY_STORE = 'categories';
+
+interface StoreNames {
+	[DAY_STORE]: Table<Day>;
+	[ACTIVITY_STORE]: Table<Activity>;
+	[CATEGORY_STORE]: Table<Category>;
+}
+
+export type StoreName = keyof StoreNames;
```

Comment:
While defining store names like this is best practice on Redux, it's a bit "unorthodox" to do in other state management libraries.
Considering that Dexie assigns properties to the class based on the table names, I think this is a good idea to use here too. However, the implementation is a bit weird.
Instead, please use TypeScript's `enum` feature to declare store names. It'd look something like this:
```
enum Stores {
  DAY = "days",
  ACTIVITY = "activities",
  CATEGORY = "category"
}
```
And the usage is: `Stores.DAY`. The file would then move to `data/enums`.